test/Spec.hs

```diff
@@ -322,3 +330,195 @@ integrationTests = testGroup "integrationTests" $
       isUbuntu = ("ubuntu" `T.isPrefixOf`)
   in map testFontset $ zip [0..] $ map T.unpack $ filter (not . isUbuntu)
      $ map fst $ uFontsets uiOptions
+
+itemKindUnitTests :: TestTree
+itemKindUnitTests = testGroup "itemKindUnitTests" $
+  let customRules = RK.RuleContent
```

Comment:
You can succinctly define this as `RK.emptyRuleContent {exceptionalField = ..., ...}`.